client/src/components/categorical/categorical.js

```diff
@@ -27,7 +21,8 @@ class Categories extends React.Component {
     this.state = {
       createAnnoModeActive: false,
       newCategoryText: "",
-      categoryToDuplicate: null
+      categoryToDuplicate: null,
+      activeSuggestItem: null
```

Comment:
this state field seems unused.  Dead code?

Reply:
Yes and no... AnnoInputs passes a function here: 

`handleItemChange={this.handleSuggestActiveItemChange}`

and we store the value, but do not currently use it. Conceivable that we would. Remove?

Reply:
I am not a fan of dead code.  If you are going to use the feature in the immediate future it is fine to leave it.  But if it is just speculative, *YAGNI*